graphs/possible_bipartition.py

```diff
@@ -1,12 +1,41 @@
-# Can be used for BFS
-from collections import deque 
+from collections import deque
+
 
 def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n * m) where n is the number of nodes/dogs 
+        and m is the number of edges
+        Space Complexity: O(n)
     """
-    pass
+    # PART 1: CREATE AN ADJACENCY LIST
+    def createGraph(nodes):
+        graph = {}
+        for index, edge in enumerate(nodes):
+            graph[index] = []
+            for neighbor in edge:
+                graph[index].append(neighbor)
+        return graph
+
+    graph = createGraph(dislikes)
```

Comment:
Totally fine to do this, I just want to note that `dislikes` is already considered an adjacency list. Adjacency lists can be represented as either a list of lists where the nodes are represented by the indices and the edges are the values(which is what we provide),  or as a dictionary where the nodes are the keys and the values are the edges.